src/main/scala/com/databricks/spark/avro/AvroRelation.scala

```diff
@@ -66,7 +107,7 @@ case class AvroRelation(location: String)(@transient val sqlContext: SQLContext)
     val fs = FileSystem.get(path.toUri, sqlContext.sparkContext.hadoopConfiguration)
 
     val status = fs.getFileStatus(path)
-    val singleFile = if (status.isDir) {
+    val singleFile = if (status.isDirectory) {
```

Comment:
isDirectory is only available in hadoop 2+. Does this project have a hard dependency on Hadoop 2+? If not, we should use isDir, despite being deprecated, to avoid breaking users of older Hadoop versions.